converter/src/main/java/gov/cms/qpp/conversion/validate/CpcMeasureDataValidator.java

```diff
@@ -64,14 +65,24 @@ private void validateAllSupplementalNodesOfSpecifiedType(
 		EnumSet<SupplementalData> codes = EnumSet.copyOf(
 				 SupplementalData.getSupplementalDataSetByType(supplementalDataType));
 
-		for (SupplementalData supplementalData : codes) {
-			Node validatedSupplementalNode = filterCorrectNode(supplementalDataNodes, supplementalData);
+		String measureId = node.getParent().getValue(QualityMeasureIdValidator.MEASURE_ID);
+		MeasureConfig config = MeasureConfigs.getConfigurationMap().get(measureId);
+		if (config != null) {
+			String electronicMeasureID = config.getElectronicMeasureId();
```

Comment:
Would it be worthwhile to add some kind of convenience functions on `MeasureConfigs`?  Make it easier to get a `MeasureConfig` without having to reference the map first?  In this code, the `MeasureConfig` isn't used beyond retrieving the `electronicMeasureID`, so having a convenience method there would be helpful and remove a few lines of code here.  There also doesn't appear to be anything going on if `config == null`.  Shouldn't that result in an error condition?

Reply:
It doesn't because measure configuration is checked and an error is added elsewhere in the code. However, we continue to validate the child nodes. If we don't check to ensure the measure configuration is there, we perform validation with a null measure configuration and get the NPE that we were seeing. I do agree that `MeasureConfig` could be separated into some kind of helper method for it. A couple of other classes use it. It would definitely increase the scope of this task a bit but it can be added here at the cost of probably not finishing at the end of this sprint (which I think is okay). @sfradkin Thoughts?

Reply:
To be honest, I think this refactor should come here because this bug is low priority. I thought because this was a bug, we might want to get this out before the that refactor.

Reply:
We need to discuss when we want to cut the next release.  If it's a fairly quick refactor, lets get it in here.  Otherwise, if we cut a release before a refactor can be done, we should get the bugfix out.

Reply:
A new story has been added for the MeasureConfigs refactor. It's not that large of a task but this bug is causing issues in production and takes priority.